app/scripts/lib/validate.js

```diff
@@ -103,7 +103,41 @@ define([
       }
 
       return password.length >= Constants.PASSWORD_MIN_LENGTH;
+    },
+
+    /**
+     * Check if a value is not falsy.
+     *
+     * @return the value if it's not falsy, the defaultValue otw.
+     */
+    isNotFalsy: function (value, defaultValue) {
```

Comment:
The function names could be modified to better convey their intent. `isNotFalsy` implies "is this value truthy?" but in reality the code chooses a value if it's value is truthy, and falls back to a default value if not. I'm trying to think up a better name, and am drawing a blank.